lib/twilio/util.ts

```diff
@@ -144,6 +144,25 @@ function flatMap(list, mapFn) {
   }, []);
 }
 
+/**
+ * Converts an EventEmitter's events into a promise and automatically
+ * cleans up handlers once the promise is resolved or rejected.
+ */
+function promisifyEvents(emitter, resolveEventName, rejectEventName) {
+  return new Promise((resolve, reject) => {
+    function resolveHandler() {
+      emitter.removeListener(rejectEventName, rejectHandler);
```

Comment:
Should we remove the resolve handler here as well? Or, when the promise is settled at all, could we just remove both at the same time?

Reply:
That's what the `.once` is for. See lines 161 and 162.

Reply:
Oops, I thought it was `.on`.